index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Felix Ouma is an experienced software developer" />
+    <title>Felix Ouma&apos;s Portfolio</title>
+    <link rel="stylesheet" href="assets/styles/style.css" />
+</head>
+<body>
+    <main>
+        <header class="flex-container">
+            <div class="logo">
+                <a href="#">Felix Ouma</a>
+            </div>
+
+            <nav>
+                <div>
+                    <img src="assets/images/Union.svg" alt="Hambuger Menu" />
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <div>
+                <h2 class="title">I’m Felix Glad to see you!</h2>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <p class="purple-text">LET'S CONNECT</p>
+
+                <div class="social-icons">
+                    <img src="assets/images/twitter.svg" alt="Twitter Logo" />
+                    <img src="assets/images/linkedinicon.svg" alt="Linked In Logo" />
+                    <img src="assets/images/Vector.svg" alt="Company Logo" />
+                    <img src="assets/images/github.svg" alt="Github Logo" />
+                    <img src="assets/images/Vector2.svg" alt="Company 2 Logo" />
+                </div>
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Using `<div>` tags to group list items is not a good use of the HTML5 semantic tag. Instead, use  a `<ul> tag` to wrap it's child elements `<li> tags`,  having each `<img> tag's` being wrapped in each `<a> tag`.